examples/micropython/main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       micropython example application
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "thread.h"
+
+#include "micropython.h"
+#include "py/stackctrl.h"
+#include "lib/utils/pyexec.h"
+
+#include "blob/boot.py.h"
+
+static char mp_heap[MP_RIOT_HEAPSIZE];
```

Comment:
Nitpick: For me the name here is not immediately clear, is there anything against simply naming it `MICROPYTHON_HEAPSIZE`? does the `mp_` prefix originate from micropython itself?

Reply:
yes, micropython uses mp_ and MP_ everywhere. I don't care which we use...

Reply:
Okay, lets keep it this way then for consistency.